### ISSUE TITLE
f.m.p doesn't generate the NodePort parameter for k8s configuration files

### ISSUE THREAD

**Issue by samolisov** [fabric8io/fabric8-maven-plugin#1140]
### Description

I get the followingg problem using fmp on Windows with minikube. So, the NodePort parameter isn't generated by fmp, therefore k8s generates new port number for every deploy. I also tried using the exposecontroller, but the controller also binds my service on new ports. The *fabric8.deploy.ignoreServices* helps 'to save' ports after the first deploy, but the question is how to get the port number for integration tests?

I'm working on the following scenario: a Java EE application is deployed on docker inside k8s and exposes a number of REST services; there are integration tests that make requests to the services and analysing the responses. Yet I must know the port the docker container is exposed on to implement the scenario. I'm sure, I'm not the first who asks so, but no answer has been googled except use Arqillian for integration tests.

### Info

* Kubernetes / OpenShift setup and version : minikube, kubernetes 1.8.0
* If it's a feature request, what is your use case : see above

**Comment by samolisov** [comment]
I've found a solution for integration tests, the following command returns the port:
```
kubectl get service <SERVICE_NAME> -n <NAMESPACE> -o jsonpath="{.spec.ports[0].nodePort}"
```
I use the `org.codehaus.gmaven:groovy-maven-plugin` maven plugin to execute the command and read the port number from the output stream. I've shared my recipe in the following blog entry: [Integration Testing a Java EE Application in the Containerized World Using Kubernetes and the Fabric8 Maven Plugin](http://psamolysov.blogspot.ru/2017/12/integration-testing-java-ee-application-inkubernetes-by-fabric8.html). Your comments are welcome!
